elasticapm/base.py

```diff
@@ -532,6 +532,11 @@ def check_python_version(self):
         v = tuple(map(int, platform.python_version_tuple()[:2]))
         if (2, 7) < v < (3, 5):
             warnings.warn("The Elastic APM agent only supports Python 2.7 and 3.5+", DeprecationWarning)
+        if v == (2, 7):
+            Client.logger.warn(
```

Comment:
Why did you go with `Client.logger.warn` instead of a `DeprecationWarning` (or probably more appropriately, a `PendingDeprecationWarning`)?

Reply:
Ooh, I didn't know about `PendingDeprecationWarning` -- I didn't want to use `DeprecationWarning` because it would be inconsistent with our current usage. I'll update to use `PendingDeprecationWarning` and add a note in the docs.